ocfweb/docs/views/officers.py

```diff
@@ -106,19 +103,21 @@ def officer_from_uid(uid):
     Term('Summer 2013', gms=['daradib'], sms=['tzhu']),
     Term('Fall 2013', gms=['daradib'], sms=['tzhu']),
     Term('Spring 2014', gms=['nickimp'], sms=['ckuehl']),
+    Term('Summer 2014', gms=['nickimp'], sms=['ckuehl']),
     Term('Fall 2014', gms=['nickimp'], sms=['ckuehl']),
     Term('Spring 2015', gms=['nickimp'], sms=['ckuehl']),
+    Term('Summer 2015', gms=['nickimp'], sms=['ckuehl']),
```

Comment:
We've never done elections for the summer, so I don't think this adds much except inflating the number of times `nickimp` and `ckuehl` appear on this page (=P)